requirements.txt

```diff
@@ -0,0 +1,26 @@
+# NVDA's build system is SCons
+SCons==4.1.0.post1
+
+# NVDA's runtime dependencies

```

Comment:
Please also add `diff_match_patch_python` so that DMP will work again on Python 3.8 and newer Python versions going forward (currently the extension in the submodule is only compatible with 3.7). Once we switch to 3.8 I'll remove the extension from the nvda_dmp repo.

Reply:
I've now opened PR #12113 (to be merged into this branch and fix DMP).